AppConnectSwift.xcodeproj/project.pbxproj

```diff
@@ -251,7 +254,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AppConnectSwift/Pods-AppConnectSwift-resources.sh\"\n";
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods/Pods-resources.sh\"\n";
```

Comment:
Not quite confident as I made some changes in Pod file to make it run on physical device that might have triggered it.